GRAPHICS/DrawBinkMovie.md

```diff
@@ -5,21 +5,34 @@ ns: GRAPHICS
 
 ```c
 // 0x7118E83EEB9F7238
-void _DRAW_BINK_MOVIE(int binkMovie, float p1, float p2, float p3, float p4, float p5, int r, int g, int b, int a);
+void _DRAW_BINK_MOVIE(int binkMovie, float posX, float posY, float scaleX, float scaleY, float rotation, int r, int g, int b, int a);
 ```
 
-```
-NativeDB Introduced: v1290
-```
+Must be called each frame, will play at specified position on screen when called with _PLAY_BINK_MOVIE
 
 ## Parameters
-* **binkMovie**:
-* **p1**:
-* **p2**:
-* **p3**:
-* **p4**:
-* **p5**:
-* **r**:
-* **g**:
-* **b**:
-* **a**:
+* **binkMovie**: The movie that is being played.
+* **posX**: The centered x position of the movie.
```

Comment:
Sorry I forgot to mention earlier, albeit obvious, perhaps it should be specified that this is a percentage that ranges from 0.0 to 1.0.

Just a quick nitpick really.

Otherwise this pretty much looks good to me in its current state 😃